sdks/python/gen_protos.py

```diff
@@ -111,6 +111,15 @@ def generate_proto_files(force=False):
             '%s' % ret_code)
 
 
+    if sys.version_info[0] >= 3:
+      ret_code = subprocess.call(
+        ["futurize", "-0", "-wv", "--no-diff", out_dir])
```

Comment:
Could you expand the arguments to their long form: `futurize --both-stages --write --verbose --no-diff` ?

Reply:
sure